### ISSUE TITLE
"No YubiKey detected" fedora repo

### ISSUE THREAD

**Issue by tmdag** [Yubico/yubioath-desktop#324]
- Yubico Authenticator version: 
4.3.3-2.fc29 (dnf install from fedora repository)

- Operating system and version: 
4.19.10-300.fc29.x86_64

- YubiKey model and version: 
YubiKey 5 NFC

- Bug description summary: 
yubioath-desktop not detecting yubikey.
Yubikey Personalization tool detects it without issue at the same time.

>  yubioath-desktop --log-level DEBUG
Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
qrc:/qml/SetPassword.qml:23:5: QML ColumnLayout: Detected anchors on an item that is managed by a layout. This is undefined behavior; use Layout.alignment instead.
qrc:/qml/SettingsDialog.qml:20:5: QML ColumnLayout: Detected anchors on an item that is managed by a layout. This is undefined behavior; use Layout.alignment instead.
qrc:/qml/AddCredentialSlot.qml:14:5: QML ColumnLayout: Detected anchors on an item that is managed by a layout. This is undefined behavior; use Layout.alignment instead.
qrc:/qml/AddCredential.qml:15:5: QML ColumnLayout: Detected anchors on an item that is managed by a layout. This is undefined behavior; use Layout.alignment instead.
qrc:/qml/main.qml:639: TypeError: Cannot read property 'code' of null

**Comment by dagheyman** [comment]
Thanks for the report. Closing as duplicate of #318

**Comment by gbcox** [comment]
I'm currently working on updating yubikey-manager and yubioath-desktop for Fedora 28 and 29.  Right now I'm waiting on a dependency - python-fido2 to become available in the Fedora repositories.  Once that is complete, I can build and make it available in the updates-testing repository - then production.  Here is the link for the python-fido2 updates:  https://bodhi.fedoraproject.org/updates/?search=fido2
